src/__test__/calculate.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import Calculator from '../components/Calculator';
+
+test('renders calculator', () => {
+  render(<Calculator />);
+  const displays = screen.getAllByText('0');
+  expect(displays.length).toBeGreaterThanOrEqual(1);
+});
+
+test('clicking AC button resets the display to "0"', () => {
+  render(<Calculator />);
+  fireEvent.click(screen.getByText('AC'));
+  // Check if at least one display resets to "0"
+  const displays = screen.getAllByText('0');
+  expect(displays.length).toBeGreaterThanOrEqual(1);
+});
```

Comment:
- [ ] Hi @Hsuwaddyhtat22, the string in the **test** block should be something that should actually happen with your tests. I think the matcher **toBeGreaterThanOrEqual** desn't assert the hypothesis you made. I think it was preferable to use the `describe` block which wrap the `test` or `it` blocks and it is the best way to group our tests. Clicking on "AC", we expect the output to be zero :+1: